File: selenium/src/java/com/thoughtworks/selenium/internal/AltLookupStrategy.java
package com.thoughtworks.selenium.internal;

import com.thoughtworks.webdriver.WebDriver;
import com.thoughtworks.webdriver.WebElement;
import com.thoughtworks.webdriver.RenderedWebElement;
import com.thoughtworks.webdriver.By;

public class AltLookupStrategy implements LookupStrategy {
    public WebElement find(WebDriver driver, String use) {
        return driver.findElement(By.xpath("//*[@alt='" + use + "']"));
    }
}

File: selenium/src/csharp/WebdriverBackedSelenium/Internal/SeleniumEmulation/LookupStrategies/AltLookupStrategy.cs

using System;
using OpenQA.Selenium;

namespace Selenium
{


    public class AltLookupStrategy : ILookupStrategy
    {
        public OpenQA.Selenium.IWebElement Find(IWebDriver driver, string use)
        {
            return driver.FindElement(By.XPath("//*[@alt='" + use + "']"));
        }
    }
}
